contracts/Gate.sol

```diff
@@ -366,20 +362,20 @@ contract Gate is ProtocolVersioned, Owned {
         // check if the stake request was accepted
         require(stakeRequest.hashLock != bytes32(0));
 
-        bytes32 uuidR = bytes32(0);
         uint256 amountST = uint256(0);
         address stakerAddress = address(0);
-        (uuidR, amountST, stakerAddress) = OpenSTValueInterface(openSTProtocol).revertStaking(_stakingIntentHash);
+        (, amountST, stakerAddress) = OpenSTValueInterface(openSTProtocol).revertStaking(_stakingIntentHash);
```

Comment:
I think you can define `address stakerAddress` here, so I think you can delete the line above.

Reply:
Moving to inline, gives compilation error.